azure/services/scalesets/scalesets.go

```diff
@@ -564,11 +564,15 @@ func (s *Service) buildVMSSFromSpec(ctx context.Context, vmssSpec azure.ScaleSet
 			nicConfig := compute.VirtualMachineScaleSetNetworkConfiguration{}
 			nicConfig.VirtualMachineScaleSetNetworkConfigurationProperties = &compute.VirtualMachineScaleSetNetworkConfigurationProperties{}
 			nicConfig.Name = pointer.String(vmssSpec.Name + "-" + strconv.Itoa(i))
-			if pointer.BoolDeref(n.AcceleratedNetworking, false) {
```

Comment:
thanks for pointing it out - a bit hard to read if you're not used to that output otherwise :)

fixed it and code is slightly easier to read anyway.